crates/services/gas_price_service/src/fuel_gas_price_updater.rs

```diff
@@ -0,0 +1,354 @@
+use crate::{
```

Comment:
Do we need changes from this file? Looks like @rymnc already migrated them and V1 will define own updater/service?